resources/original_messages.json

```diff
@@ -77,6 +77,7 @@
   },
   "contact_view_exit_connection": {
     "description": "Message shown to users who are trying to contact support about an unsupported issue.",
+    "meaning": "Message shown to users who are trying to contact support about an unsupported issue.",
```

Comment:
Shall we simplify use something like `MsgID-20240319-UpdateWhenPlaceHolderChanged` instead of duplicating the `description`? Because maybe the duplicated fields will confuse people? For example, when they update the "description", will they need to update the `meaning` as well?

Reply:
Sure. Done!